### PR TITLE
Add vrrp virtual address

### PR DESCRIPTION
**Pull request by sbarguil**
Virtual IPv4 addresses for a single VRRP group.

### CODE REVIEW

File: yang/ietf-l3vpn-ntw.yang [hunk 1]
```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
                         description
                           "Indicates the IP address of the peer.";
                       }
+                      leaf virtual-ip-address {
```
Comment (boucadair) [inline yang/ietf-l3vpn-ntw.yang:2317]: To be defined as a leaf-list

Note that this is defined as list virtual-ipv4-address and list virtual-ipv6-address in rfc8347, but I think that a leaf-list is more approrpiate.

File: yang/ietf-l3vpn-ntw.yang [hunk 2]
```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
...
+                        type inet:ip-address;
+                        description
+                          "Virtual IPv4 addresses for a single VRRP group";
+                        reference
```
Comment (boucadair) [inline yang/ietf-l3vpn-ntw.yang:2321]: No need to have an IPv4-specific reference as the type is inet:ip-address. I suggest to remove the reference statement.

**Review by boucadair — CHANGES_REQUESTED** [review]

File: yang/ietf-l3vpn-ntw.yang [hunk 3]
```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
                         description
                           "Indicates the IP address of the peer.";
                       }
+                      leaf virtual-ip-address {
+                        type inet:ip-address;
+                        description
+                          "Virtual IPv4 addresses for a single VRRP group";
```
Comment (sbarguil) [inline yang/ietf-l3vpn-ntw.yang:2320]:
```suggestion
                          "Virtual addresses for a single VRRP group";
```

File: yang/ietf-l3vpn-ntw.yang (hunk 1 above)
Comment (sbarguil) [inline yang/ietf-l3vpn-ntw.yang:2317]:
```suggestion
                      leaf-list virtual-ip-address {
```

File: yang/ietf-l3vpn-ntw.yang [hunk 4]
```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
                         description
                           "Indicates the IP address of the peer.";
                       }
+                      leaf-list virtual-ip-address {
+                        type inet:ip-address;
+                        description
+                          "Virtual addresses for a single VRRP group";
```
Comment (boucadair) [inline yang/ietf-l3vpn-ntw.yang:2320]:
```suggestion
                          "Virtual IP addresses for a single VRRP group. ";
```

File: yang/ietf-l3vpn-ntw.yang [hunk 5]
```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
...
+                        reference
+                         "RFC 5798: Virtual Router Redundancy Protocol (VRRP)
+                          Version 3 for IPv4 and IPv6.  Section 1.2";
```
Comment (boucadair) [inline yang/ietf-l3vpn-ntw.yang:2323]:
```suggestion
                          Version 3 for IPv4 and IPv6,  Sections 1.2 and 1.3";
```

**Pull request merged**